src/licensedcode/data/licenses/lua.LICENSE

```diff
@@ -0,0 +1,6 @@
+Copyright © 1994–2019 Lua.org, PUC-Rio.
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
This is exactly an MIT license ... so why would this warrant to be its new license record?